inspectit-ocelot-core/src/main/java/rocks/inspectit/ocelot/core/instrumentation/config/InstrumentationConfigurationResolver.java

```diff
@@ -277,7 +277,7 @@ boolean isIgnoredClass(TypeDescriptionWithClassLoader typeWithLoader, Instrument
             return true;
         }
 
-        if (config.getSource().isExcludeLambdas() && type.getName().contains("$$Lambda$")) {
+        if (config.getSource().isExcludeLambdas() && (type.getName().contains("$$Lambda$") || type.getName().contains("$$Lambda"))) {
```

Comment:
Das ODER kann weg und dann brauchen wir nur contains("$$Lambda") hinterm UND.
Jedes "$$Lambda$" wird ja auch durch "$$Lambda" erfüllt.

Reply:
Agree, ist mir selbst auch direkt nach dem Commit in den Kopf gekommen.